system-container/README.md

```diff
@@ -0,0 +1,13 @@
+# System container installer
+
+These files are needed to run the installer using an [Atomic System container](http://www.projectatomic.io/blog/2016/09/intro-to-system-containers/).
+
+* config.json.template - It is the configuration file for running the runC container.
```

Comment:
This is a template of the config and not the config file itself ;)

Perhaps we could drop "runC" and say simply container. For someone who is not versed in all of the names and concepts, it can be confusing to see "system-container", "Atomic System Container" and "runC container" all in one place.

Suggestion:

```markdown
* `config.json.template` - Template of the configuration file used for running containers.
```